examples/cifar_ViT.py

```diff
@@ -205,33 +203,15 @@ def test_step(self, batch, _):
     NUM_WORKERS = 4
     GPUS = 1
 
-    train_transforms = transforms.Compose(
```

Comment:
same as above, these are actually the default transforms in the lightning bolt datamodule, not useful